manifest: update sdk-connectedhomeip to bring matter fix (#12291)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/known_issues.rst

The issues in this section are related to the :ref:`ug_matter` protocol.

> .. rst-class:: v2-4-1 v2-4-0

Reviewer: should v2.4.2 be added here as well?

Author: right, @ArekBalysNordic: Does this known issue (KRKNWK-17535)  need update too?

> KRKNWK-17699: Non complaiant Basic Information cluster revision for Matter 1.1.0. The issue result in fail at TC-BINFO-1.1 Matter Certifiction test.

Reviewer 2: ```suggestion
KRKNWK-17699: Failure at TC-BINFO-1.1 Matter Certification test.
  The issue happens due to a noncompliant Basic Information cluster revision for Matter 1.1.0.
```

> KRKNWK-17699: Non complaiant Basic Information cluster revision for Matter 1.1.0. The issue result in fail at TC-BINFO-1.1 Matter Certifiction test.

Reviewer 3: ```suggestion
KRKNWK-17699: Non complaiant Basic Information cluster revision for Matter 1.1.0. The issue results in fail at TC-BINFO-1.1 Matter Certifiction test.
```

>   **Workaround:** Set the default value of the Basic Information cluster revision to 1 in ZAP and regenerate files.

Reviewer 3: I think we should be more explicit here: 
- point directly to the file which needs to be updated
- mention how the files can be regenerated (I guess saying "with ZAP Tool" would suffice)

>   **Workaround:** Set the default value of the Basic Information cluster revision to 1 in ZAP and regenerate files.

Reviewer 2: Does "in ZAP" here mean "using the ZAP tool" or "in a ZAP file"? And if it is the latter, is this a specific file where the change needs to be made?

.. rst-class:: v2-4-1 v2-4-0 v2-3-0 v2-2-0

> KRKNWK-17594: Application may not response for the forcing arm-fail-safe cleanup due to unecessery reset Thread interface. The impact may result in fail at TC-CNET-4.10 Matter Certifiction test.

Reviewer 3: ```suggestion
KRKNWK-17594: Application may not response for forcing of the arm-fail-safe cleanup due to unnecessary reset of the Thread interface. The impact may result in fail at TC-CNET-4.10 Matter Certification test.
```
@wiba-nordic  please verify :)

> KRKNWK-17594: Application may not response for the forcing arm-fail-safe cleanup due to unecessery reset Thread interface. The impact may result in fail at TC-CNET-4.10 Matter Certifiction test.

Reviewer 2: ```suggestion
KRKNWK-17594: Application does not always respond when forcing arm-fail-safe cleanup because of the Thread interface being unnecessarily reset.
  This can result in the TC-CNET-4.10 Matter Certification test failing.
```
A couple of points:
* What is "arm-fail-safe cleanup"? It sounds like something ARM-based has failed a safe cleanup.
* The way I worded it means that the forcing the cleanup happens because the Thread interface was reset without reason, but it could also be that it should be an unexpected reset of the Thread interface or that the not responding happens due the unnecessary reset of the Thread interface.

Author: * arm-fail-safe is an procedure defined by Matter Specification.
* not responding happens due to unnecessary reset of the Thread Interface

Reviewer 2: ```suggestion
KRKNWK-17594: Application does not always respond when forcing fail-safe cleanup.
  This happens because of the Thread interface being unnecessarily reset, and can result in the TC-CNET-4.10 Matter Certification test failing.
```

The issues in this section are related to the :ref:`ug_matter` protocol.

.. rst-class:: v2-4-2 v2-4-1 v2-4-0

> KRKNWK-17699: Failure at TC-BINFO-1.1 Matter Certification test for lock sample.
>   The issue happens due to a noncompliant Basic Information cluster revision for Matter 1.1.0 in lock sample.

Reviewer 2: ```suggestion
KRKNWK-17699: Failure at TC-BINFO-1.1 Matter Certification test for the lock sample.
  The issue happens due to a noncompliant Basic Information cluster revision for Matter 1.1.0 in the lock sample.
```

>   **Workaround:** Set the default value of the Basic Information cluster revision to 1 in samples/matter/lock/src/lock.zap by ZAP Tool and regenerate files.

Reviewer 2: ```suggestion
  **Workaround:** Set the default value of the Basic Information cluster revision to 1 in :file:`samples/matter/lock/src/lock.zap` using the ZAP Tool and regenerate files.
```

.. rst-class:: v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0

KRKNWK-17594: Application does not always respond when forcing fail-safe cleanup.

>   The lack of response can happen because of the Thread interface being unnecessarily reset.
>   This can result in the TC-CNET-4.10 Matter Certification test failing.

Reviewer 2: ```suggestion
  This can happen because of the Thread interface being unnecessarily reset and can result in the TC-CNET-4.10 Matter Certification test failing.
```
I think this reads just a little bit better. Sorry for going back and forth.
